src/components/services/Events.vue

```diff
@@ -1,39 +1,113 @@
+<script>
+  import CustomHeader from '@/components/shared/CustomHeader.vue';
+  export default {
+    components: {
+      CustomHeader,
+    },
+    data() {
+       return {
+          items: [
+            {message: 'La Scala Opera Trip'},
+            {message: 'Night Club Hop'}
+          ]
+       }
+   }
+  }
+</script>
+
 <template>
   <div class="events-home">
-    <h1>Events Home</h1>
-    <h3 class="greetings">{{ msg }}</h3>
+    <CustomHeader title="My Concierge" subtitle="Special Events"/>
+
+    <nav>
+      <router-link to="/">Home</router-link>
+      <router-link to="/calendar">Calendar</router-link>
+      <router-link to="/services">Services</router-link>
+    </nav>
+
+    <main>
+      <div class="left-column">
+        <div class="reservation-box">
+          <span>placeholder for reservation form</span>
+        </div>
+        <div class="contact-box">
+          <span>Contact Special Events: 555-222-7878</span>
+        </div>
+      </div>
+      <div class="right-column">
+        <div class="events-box">
+          <h3>Upcoming Events</h3>
+          <li v-for = "(item, index) in items">
```

Comment:
Update these v-for items to display all the data that's shown in the Figma design. To do that, I would suggest changing these from 'li' to 'div' because that will make it easer to style a multi-line section including a header and four lines beneath that.